feature/platform/transceiver/ZR_pre-fec_ber_test/README.md

```diff
@@ -0,0 +1,49 @@
+# gNMI-1.20: Telemetry: 400ZR Optics Pre-FEC(Forward Error Correction) BER(Bit Error Rate)
+
+## Summary
+
+Validate 400ZR optics module reports pre-FEC bit error rate performance data.
+
+## Procedure
+
+*   Connect two ZR interfaces using a duplex LC fiber jumper such that TX
+    output power of one is the RX input power of the other module.
+*   To establish a point to point ZR link ensure the following:
+      * Both transceivers state is enabled
+      * Both transceivers are set to a valid target TX output power
+        example -10 dBm
+      * Both transceivers are tuned to a valid centre frequency
+        example 193.1 THz
+*   With the link ZR link established as explained above, verify that the
+    following ZR transceiver telemetry paths exist and are streamed for both
+    the ZR optics
+    *   /terminal-device/logical-channels/channel/otn/state/pre-fec-ber/instant
+    *   /terminal-device/logical-channels/channel/otn/state/pre-fec-ber/avg
+    *   /terminal-device/logical-channels/channel/otn/state/pre-fec-ber/min
+    *   /terminal-device/logical-channels/channel/otn/state/pre-fec-ber/max
+
+**Note:** For min max avg values, performance monitoring bin of 10 sec is
+          preferred. These values should be retrieved using a gNMI.Subscribe
+          with a SAMPLE subscription.
+
+
+*   Verify that the optics pre-FEC BER is updated after the interface is
```

Comment:
grammar nit

 Verify that the optics pre-FEC BER is updated after the interface flaps.

Reply:
Done